packages/nextjs/generated/deployedContracts.ts

```diff
@@ -1,3 +1,15 @@
-const deployedContracts = null;
+import { GenericContractsDeclaration } from "~~/utils/scaffold-eth/contract";
+
+/**
+ * @example
+ * const deployedContracts = {
+ *   31337: {
+ *    YourContract: {
+ *      address: "0x..."
+ *      abi: [...],
+ *    }
+ * } satisfies GenericContractsDeclaration | null;
+ */
+const deployedContracts = null satisfies GenericContractsDeclaration | null;
 
 export default deployedContracts;
```

Comment:
Could we do this? (I think is valid)

```ts
const deployedContracts = null;

export default deployedContracts satisfies GenericContractsDeclaration | null;
```

I think it's more legible like this, right?

We'd need to change `99_generateTsAbis.ts` + change the example.